plugin-cgroupv1/src/oar2/probe.rs

```diff
@@ -1,26 +1,38 @@
 use alumet::{
-    measurement::{MeasurementAccumulator, Timestamp},
-    metrics::TypedMetricId,
+    measurement::{AttributeValue, MeasurementAccumulator, MeasurementPoint, Timestamp},
     pipeline::{elements::error::PollError, Source},
-    plugin::util::CounterDiff,
 };
-use std::result::Result::Ok;
+use std::{collections::HashMap, result::Result::Ok};
 
-use super::utils::OpenedCgroupv1;
-
-use crate::cgroupv1::gather_value;
+use crate::cgroupv1::{Cgroupv1Probe, Metrics};
 
 pub struct OAR2Probe {
```

Comment:
```suggestion
pub struct Oar2Probe {
```
The struct should be renamed to adhere to Rust naming conventions.